docs/tips-and-tricks.md

```diff
@@ -30,42 +48,53 @@ Use this snippet at the top of your userstyle to import and use the font:
 
 ### How do I convert preview images to WebP?
 
-The Catppuccin project prefers to use the [WebP image format](https://en.wikipedia.org/wiki/WebP) for the asset preview images. We recommend using Google's [`cwebp`](https://developers.google.com/speed/webp/docs/cwebp) conversion utility to convert images to WebP.
+The Catppuccin project prefers to use the
+[WebP image format](https://en.wikipedia.org/wiki/WebP) for the asset preview
+images. We recommend using Google's
+[`cwebp`](https://developers.google.com/speed/webp/docs/cwebp) conversion
+utility to convert images to WebP.
 
 #### Installation
 
 | Method                                | Instructions                                                                           |
 | ------------------------------------- | -------------------------------------------------------------------------------------- |
-| [Homebrew](https://brew.sh/)          | `brew install webp` [[view cask]](https://formulae.brew.sh/formula/webp)               |
-| [MacPorts](https://www.macports.org/) | `sudo port install webp` [[view port]](https://ports.macports.org/port/webp/)          |
```

Comment:
Removing these brackets is not just a formatting change, it changes the appearance.